src/storage/mod.rs

```diff
@@ -206,6 +207,9 @@ impl Display for Command {
             Command::RawGet { ref ctx, ref key } => {
                 write!(f, "kv::command::rawget {:?} | {:?}", key, ctx)
             }
+            Command::Pause { ref ctx, duration } => {
```

Comment:
As I and jay have discussed, we will use future and no need add ugly `#[cfg(test)]`.